src/ChunkMap.cpp

```diff
@@ -791,6 +791,28 @@ bool cChunkMap::DoWithChunk(int a_ChunkX, int a_ChunkZ, cChunkCallback & a_Callb
 }
 
 
+bool cChunkMap::DoWithChunkAt(Vector3i a_Pos, std::function<bool(cChunk&)> a_Callback)
```

Comment:
Also, is `a_Pos` in block coords or chunk coords? If in block coords, we customarily name it `a_BlockPos`.